### ISSUE TITLE
Spanish Translation issue

### ISSUE THREAD

**Issue by Danixd12** [revoltchat/revite#308]
Channel categories configuration still in English, and I can't find the string to translate them, so I think it's a bug.
![Captura de pantalla 2021-09-26 225405](https://user-images.githubusercontent.com/41576632/134823833-f5c156bc-9de6-41b1-8383-6f029c8d401a.png)
.

**Comment by aeristhy** [comment]
Yeah, I noticed that as well. Been searching for the string but can't find it

**Comment by Rexogamer** [comment]
Not all strings have been made translatable and added to this repo yet. I shall make a PR for these once my other i18n PR(s) are merged.

**Comment by aeristhy** [comment]
cool
